deployment/src/main/resources/templates/pojo.qute

```diff
@@ -32,6 +32,7 @@ import com.fasterxml.jackson.annotation.JsonProperty;
 public class {m.classname} {#if m.parent}extends {m.parent}{/if}{#if m.serializableModel} implements Serializable{/if} {
 
     {#for v in m.vars}
+    {#if !v.deprecated || openapi:genDeprecatedModelAttr(package, m.classname, codegen)}
```

Comment:
Im wondering it it wont be better to have 
v.isDeprecated( ) method, so not using namespace method but template extension https://quarkus.io/guides/qute-reference#template_extension_methods
Yes, it will take a little bit more of investigation (to see how to configure that in the template engine without annotations) but the template will be far simple.

Reply:
Actually, I think the former approach was better when I implemented the `ValueResolver`. This way we won't need to send parameters. I'd rather have a namespaced extension available in the `openapi` namespace. This way there's a clear separation of concerns of the functions we are using there.